spec/javascripts/app/views/help_view_spec.js

```diff
@@ -134,4 +141,28 @@ describe("app.views.Help", function(){
       });
     });
   });
+
+  describe("chat section", function(){
+    describe("chat enabled", function(){
+      beforeEach(function(){
+        gon.chatEnabled = true;
```

Comment:
Yes it changes because `gon` changes snake_case to headlessCamelCase. This is awkward but meh... I should change the help view instead.